cli/tests/unit/text_encoding_test.ts

```diff
@@ -50,6 +50,16 @@ unitTest(function atobThrows2(): void {
   assert(threw);
 });
 
+unitTest(function atobThrows3(): void {
+  let threw = true;
+  try {
+    atob("foobar!!");
+  } catch (DOMException) {
```

Comment:
Hi, @gorogoroumaru, @kitsonk

What about following approach?

```javascript
let threw = true;
try {
  ...
} catch (e) {
  if (e instanceof DOMException) {
    threw = false;
  }
}
assert(threw);
```

Reply:
Thank you @jaeheonji!
Looks good!
@kitsonk, how about you?